streaming/src/main/scala/org/apache/carbondata/streaming/StreamSinkFactory.scala

```diff
@@ -67,7 +67,7 @@ object StreamSinkFactory {
       carbonLoadModel.getFactFilePath,
       false,
       parameters.asJava,
-      null,
+      parameters.asJava,
```

Comment:
add comment why we need to pass same parameters twice

Reply:
while triggering the LoadTablePreExecutionEvent we reuire options provided by user and the finalOptions . In case of streaming both are same